server/src/app.test.ts

```diff
@@ -86,8 +90,21 @@ describe('errors', () => {
 
   test('returns a generic 500 response when an endpoints throws an error', async () => {
     // calling the /api/token endpoint without proper mocking throws TypeError
+    const expectedError = { error: 'client.createUserAndToken is not a function' };
     const getResponse = await request(app).get('/api/token');
     expect(getResponse.status).toEqual(500);
-    expect(getResponse.text).toEqual(JSON.stringify(ERROR_PAYLOAD_500));
+    expect(getResponse.text).toEqual(JSON.stringify(expectedError));
+  });
+
+  test('return 200 sending option request', async () => {
+    const getResponse = await request(app).options('/api/surveyReesult');
+
+    expect(getResponse.status).toBe(200);
+  });
+
+  test('check if /api/surveyResults route is open without cosmosDb configs', async () => {
```

Comment:
It is not clear how the cosmosDb is set to undefined here. If I set my CosmosDb connection string env variable, will this fail?

I see there is a related test case in `surveyRoute.test.ts`, shall we move this one there as well? That way we have a way of explicitly setting `cosmosDb` to undefined.